bitnami/fluentd/README.md

```diff
@@ -135,6 +135,8 @@ The following tables lists the configurable parameters of the kibana chart and t
 | `aggregator.nodeSelector`                       | Node labels for pod assignment                                                                                 | `{}`                                                                                                    |
 | `aggregator.tolerations`                        | Tolerations for pod assignment                                                                                 | `[]`                                                                                                    |
 | `aggregator.affinity`                           | Affinity for pod assignment                                                                                    | `{}`                                                                                                    |
+| `aggregator.extraVolumes`                       | Extra volumes                                                                                                  | `nil`                                                                                                    |
+| `aggregator.extraVolumeMounts`                  | Mount extra volume(s),                                                                                         | `nil`                                                                                                    |
```

Comment:
Shouldn't it be placed in the forwarder section as `forwarder.*`?

Reply:
Yes I fixed the typo